polyfills/document/elementsFromPoint/polyfill.js

```diff
@@ -0,0 +1,63 @@
+document.elementsFromPoint = document.msElementsFromPoint || function elementsFromPoint(x, y) {
+    var stack = [];
+    var element = document.elementFromPoint(x, y);
```

Comment:
```suggestion
document.elementsFromPoint = function elementsFromPoint(x, y) {
    if ('msElementsFromPoint' in this) {
        return this.msElementsFromPoint(x, y) || [];
    }

    var stack = [];
    var element = document.elementFromPoint(x, y);
```

`msElementsFromPoint` returns `null` instead of an empty array

Reply:
This can also be tested for in `detect.js`

```
(function () {
	if (!('document' in self && 'elementsFromPoint' in self.document)) {
		return false;
	}

	try {
		var res = document.elementsFromPoint(-9007199254740991, 9007199254740991);
		if (!res) {
			return false;
		}
	} catch (_) {
		return false;
	}

	return true;
})()
```

Reply:
I 've integrated your suggestion, but not the latter as it is quite and edge case and should be happening only on IE where is handled by the first.